### PR TITLE
Add an MRN layer and adjust layer help

### PR DESCRIPTION
**Pull request by dabreegster**
Add a layer for the Major Road Network, generated in https://github.com/acteng/atip-data-prep/pull/14.

Also switch from tooltips on the checkboxes to a full help modal. That lets us include external links, full attribution, and -- for some of the next layers -- give more details about assumptions / limitations of the data.

https://github.com/acteng/atip/assets/1664407/9e6a6cc3-5381-4188-a5f6-109ba55862bf

### CODE REVIEW

File: src/lib/browse/MrnLayer.svelte
Comment (dabreegster) [inline src/lib/browse/MrnLayer.svelte:1]: OptionalLayer.svelte is a bit of a misnomer -- it's just for simple colored polygon amenities, with a certain fixed structure around the naming of everything. A bit of the logic is copied here. I thought about refactoring some pieces of it, but nothing jumps out as useful yet. As we add more layers, I'm sure patterns will emerge.

**Comment by robinlovelace-ate** [comment]
Based on the vid, the lines are really wide, can they pls be thinner? Other than that big 👍

**Review by robinlovelace-ate — APPROVED** [review]
Many people have been asking for this, great to see it in ATIP!

File: src/lib/browse/MrnLayer.svelte [hunk 1]
```diff
@@ -26,7 +26,7 @@
     source: name,
     sourceLayer: name,
     color,
-    width: 10,
+    width: 7,
```
Comment (robinlovelace-ate) [inline src/lib/browse/MrnLayer.svelte:29]: 👍

**Review by robinlovelace-ate — APPROVED** [review]

**Pull request merged**